ykrt/src/mt.rs

```diff
@@ -28,10 +28,11 @@ const PHASE_COMPILED: usize = 0b000; // Value is a pointer to a chunk of memory
                                      // CompiledTrace<I>.
 const PHASE_TRACING: usize = 0b001; // Value is a pointer to an Arc<ThreadIdInner> representing a
                                     // thread ID.
-const PHASE_COMPILING: usize = 0b010; // Value is a pointer to a `Box<CompilingTrace<I>>`.
-const PHASE_COUNTING: usize = 0b011; // Value specifies how many times we've seen this Location.
-const PHASE_LOCKED: usize = 0b100; // No associated value.
-const PHASE_DONT_TRACE: usize = 0b101; // No associated value.
+const PHASE_TRACING_LOCK: usize = 0b010; // No associated value
```

Comment:
Should we make all of these descriptions line up to appease my OCD?

Reply:
`cargo fmt` destroys any such attempt -- I've tried :/

Reply:
The alternative is to move the comments before the `const`. That would also allow them to be doc-comments.